src/components/Fund/FundBanner.tsx

```diff
@@ -1,106 +1,161 @@
-      <img className={styles.vectorIcon16} alt="" src="../locofy/vector6.svg" />
-      <div className={styles.groupDiv11}>
-        <div className={styles.groupDiv12}>
-          <div className={styles.groupDiv12}>
-            <div className={styles.groupDiv14}>
-              <div className={styles.groupDiv14}>
-                <div className={styles.shareDiv}>{`Share `}</div>
-                <img
-                  className={styles.groupIcon3}
-                  alt=""
-                  src="../locofy/group-2376731.svg"
-                />
-              </div>
+    <div className='container mx-auto my-20'>
+        <div className='bg-[#1c1b25] pt-10 px-8 rounded-xl mx-5'>
+            <div className='grid grid-cols-2'>
+                <div className='space-y-6'>
+                    <ListBox
+                      selected={selected}
+                      setSelected={setSelected}
+                      funds={funds}
+                    ></ListBox>
+                    <div className='flex space-x-12'>
+                        <p className='text-xs font-medium text-gray-400'>
+                            <Trans>
+                                Expiry in:
+                            </Trans>
+                            <span className='bg-[#33333f] py-1 px-2 rounded-xl ml-2'>
+                                <Trans>
+                                    {expiresIn} days
+                                </Trans>
+                            </span>
+                        </p>
+                        <p className='text-xs font-medium text-gray-400'>
+                            <Trans>
+                                Investors:
+                            </Trans>
+                            <span className='bg-[#33333f] py-1 px-2 rounded-xl ml-2'>
+                                <Trans>
+                                    {investors}
+                                </Trans> 
+                            </span>
+                        </p>
+                    </div>
+                    <div className='flex space-x-3'>
+                        <p className='text-xs font-medium text-gray-400 mr-6'>{walletAddress}</p>
+                        <button className='cursor-pointer'>
+                            <img src={copyIcon} alt="" />
+                        </button>
+                        <button className='cursor-pointer'>
+                            <img src={qrCodeIcon} alt="" />
+                        </button>
+                    </div>
+                </div>
+                <div>
+                    <div className='flex justify-between'>
+                        <p className='text-3xl text-white font-light'>
+                            <Trans>
+                                Portfolio Value
+                            </Trans>
+                        </p>
+                        <div className='space-x-2'>
+                            <button
```

Comment:
we dont need this 24H 1D etc. It's attached to the little graph, and since we dont have that graph, wecan remov ethis.

Reply:
Okay Great